lib/workers/pr/index.ts

```diff
@@ -37,6 +37,11 @@ async function addCodeOwners(
   return [...new Set(assigneesOrReviewers.concat(await codeOwnersForPr(pr)))];
 }
 
+function filterUnavailableUsers(users: string[]): string[] {
+  return users;
+  // TODO: Delegate filtering logic to platform.
```

Comment:
I'd imagine a new method at the platform API here. For GitLab it would filter based on the `availability` of the users, on other platforms it would just return the input collection.

Reply:
Alternatively we could add something like `isUserAvailable` at the platform level and filter the set here.

Reply:
probably best to add `platform.filterUnavailableUsers()` so that if platforms can optimize that better than O(n) for n users then they can.

Reply:
I think this ca be an optional platform function, so only supported platforms need to implement it.